README.md

```diff
@@ -22,6 +22,11 @@ XRAY_VIVADO_SETTINGS to point to the settings64.sh file of the installed vivado
 
     export XRAY_VIVADO_SETTINGS=/opt/Xilinx/Vivado/2017.2/settings64.sh
 
+Do not source the settings64.sh in your shell, since this adds directories of
+the Vivado installation at the beginning of your PATH and LD_LIBRARY_PATH
+variables, which will likely interfere with or break non-Vivado applications in
+that shell.
```

Comment:
Add a note that it will be sourced automatically by prjxray, otherwise LGTM

Reply:
Probably worth mentioning $XRAY_VIVADO env var that is defined after sourcing utils/environment.sh, which will invoke Vivado after sourcing the settings file